cli/valet.php

```diff
@@ -289,6 +289,61 @@
     })->descriptions('Change the version of php used by valet', [
         'phpVersion' => 'The PHP version you want to use, e.g php@7.2',
     ]);
+
+    /**
+     * Tail log files.
+     */
+    $app->command('logs [-f|--follow] [-l|--lines=] [key]', function ($follow, $lines, $key = null) {
```

Comment:
I know I originally said "logs", but what would you think if this said "log" instead of "logs"? I'm not 100% sure but definitely considering `log`

Reply:
I love that Laravel/Symfony console apps can partially match command names.

If you decided to leave the definition as `logs`, you'd still be able to call it via `log`:

![logs-vs-log](https://user-images.githubusercontent.com/1914481/56010949-f743c700-5d28-11e9-8466-c700d5da810f.png)

I'll leave the decision up to you!

🤓👍

Reply:
True. I do like that.

Reply:
I also wonder if `tail` is the better command, since that's what it's actually doing. And then we could add `logs` as an alias (like `domain` is an alias for `tld`).

But I'll defer to @mattstauffer for final direction on this.

Reply:
I preferred `tail` for a bit, but asked around a bit and got a pretty overwhelming consensus that, while `tail` is more accurate, `log` or `logs` are more broadly useful--folks familiar with `tail` will get it, but folks unfamiliar with `tail` will get it *much better*.

Reply:
OK. My preference is `log`. :) Thanks for working with me!

Reply:
Done! (12ca66ade13597c1a8bcc6f485e5a91a6e3fdc7a)